package.json

```diff
@@ -8,6 +8,7 @@
   },
   "scripts": {
     "start": "NODE_ENV=development gulp start:dev",
+    "start:dev": "ng serve --project=ng-zorro-antd-doc --configuration=development",
```

Comment:
This is not the right way to add a development model.

In fact, you should modify the scripts/gulp/tasks file to find `serve:site` and add the `--configuration=development` parameter:

```diff
/** Run `ng serve` */
task('serve:site', done => {
  detectPort(4200).then((port: number) => {
    execNodeTask('@angular/cli', 'ng', [
      'serve',
      '--port',
      port === 4200 ? '4200' : '0',
      '--project=ng-zorro-antd-doc',
+      '--configuration=development'
    ])(done);
  });
});
```